src/main/resources/templates/messages.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <link href="/static/main.css" rel="stylesheet"/>
```

Comment:
<link th:href="@{/styles/main.css}" rel="stylesheet"/>
After putting the css file in a folder called styles, otherwise the connection between the stylsheet and the html file doesn't seem to be working when using Thymeleaf.